### ISSUE TITLE
catch(swal.noop) and icons animation disabling

### ISSUE THREAD

**Issue by ApolloCr** [limonte/sweetalert2#551]
hi

from point of view of your regular user:

1. please disable need of write after every swal line ".catch(swal.noop)", it is just more complicated, who wants handle dismissals will do it even without these errors to console

2. give us option to disable icons animation - few linues of code for you, great asset for us

thank you

**Comment by limonte** [comment]
Hello,

> please disable need of write after every swal line ".catch(swal.noop)", it is just more complicated, who wants handle dismissals will do it even without these errors to console

https://github.com/limonte/sweetalert2/issues/540

> give us option to disable icons animation - few linues of code for you, great asset for us

```js
swal({
  ...
  animation: false
})

**Comment by ApolloCr** [comment]
May I ask you for few more minutes?
Actually I know about 'animation' and I found bug maybe?
For me is modal not animated but success and error icons are animated, then I found this is happening when
```
swal({
  title: 'Auto close alert!',
  type: 'success',
  animation: 'false',
})
```
and everything is static with
```
swal({
  title: 'Auto close alert!',
  type: 'success',
  animation: false,
})
```

Actually this will be very useful, with this 3 levels of animation (off, icons, icons+modal), just maybe better values 'none', 'icons', 'all' and true = 'all' and false = 'none' for backwards compatibility.

thanks
